psycop/projects/scz_bp/model_training/binary_classification_pipeline.py/imblearn_binary_classification_pipeline.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+from imblearn.pipeline import Pipeline
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.task.base_metric import PredProbaSeries
+
+
+@BaselineRegistry.task_pipelines.register("imblearn_binary_classification_pipeline")
+class ImblearnBinaryClassificationPipeline:
+    def __init__(self, sklearn_pipe: Pipeline):
```

Comment:
If `Imblearn.Pipeline` is a subclass of `sklearn.Pipeline`, and does not have invalid overrides, it should be allowed in our `BinaryClassificationPipeline`. Did you get an error, or is there some other reason you needed a separate pipeline class? :-)